### ISSUE TITLE
If Markup Word Count Threshold dialog is dismissed via Window Manager button it can't be re-popped

### ISSUE THREAD

**Issue by windymilla** [DistributedProofreaders/guiguts#113]
Platform: all
Perl: all
Guiguts version: 1.1.0

To reproduce:
1. Load any file into Guiguts
2. Open Tools | Word Frequency
3. Click Ital/Bold/SC - Word Count Threshold dialog pops
4. Click the Window manager cross to dismiss (not the OK button) - dialog is dismissed
5. Click Ital/Bold/SC again - Word Count Threshold dialog does not reappear.

Check for other similar small dialogs that do not have `WM_DELETE_WINDOW` protocol defined - should operate like a "Cancel" button and at least execute
```
$::lglobal{dialogname}->destroy;
undef $::lglobal{dialogname};
```
if that is done for the OK button.

**Comment by windymilla** [comment]
Look at markuppop, and also utfentrypop, setlangpop, gotolinepop, gotopagpop, gotolabpop, versionbox

These use deiconify, probably OK: aboutpop, hotkeyspop, regexrefpop, pagelabelpop, multihelppop, pagesephelppop, marginspop, fontpop, selectionpop

Consider using killpopup() to kill popups?

**Comment by windymilla** [comment]
Addressed by #130
